vm/ops/groups.cpp

```diff
@@ -419,6 +419,67 @@ void x86CPU::op_group_D3(){
 	}
 }
 
+void x86CPU::op_group_C0(){
```

Comment:
Both groups C0 and C1 take an immediate. So, it is necessary to do `eip++` either in the group, or in each of the opcodes within the group. If you don't do eip++, then when the processor tries to execute the next instruction, it will instead execute that immediate byte

Reply:
Update the change. thank you!